### PR TITLE
Dev server

### PR DESCRIPTION
**Pull request by hathadam**
# PR Request Form

Please include a small summary of the changes you have made below: 

Updated POST Ratings and Reviews

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Unresolved issue (fix or feature that hasn't been completeted yet)

## Has This Been Tested?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: client/src/components/ratingsReviews/RatingsReviews.jsx [hunk 1]
```diff
@@ -208,19 +200,24 @@ export default function RatingsReviews() {
...
     setTimeout(() => {
       setLoaded(true);
-    }, 400);
+    }, 600);
     // get review api data
     const getReviewApi = async () => {
       try {
         const res = await axios.get(
-          `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/reviews/?count=1000&product_id=${selectedProduct.id}`,
+          'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/reviews/',
```
Comment (kenneth-bakke) [inline client/src/components/ratingsReviews/RatingsReviews.jsx:208]: switch to `http://localhost:3000/api/reviews`

File: client/src/components/ratingsReviews/RatingsReviews.jsx [hunk 2]
```diff
@@ -208,19 +200,24 @@ export default function RatingsReviews() {
...
           {
+            params: {
+              count: 1000,
+              product_id: selectedProduct.id
+            },
+
             headers: {
               Authorization: `${TOKEN}`,
             },
```
Comment (kenneth-bakke) [inline client/src/components/ratingsReviews/RatingsReviews.jsx:217]: Will be replaced with content type on server upload

File: client/src/components/ratingsReviews/RatingsReviews.jsx [hunk 3]
```diff
@@ -236,8 +233,13 @@ export default function RatingsReviews() {
...
       try {
         const res = await axios.get(
-          `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/reviews/meta/?count=50&product_id=${selectedProduct.id}`,
+          'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/reviews/meta',
           {
+            params: {
+              count: 50,
+              product_id: selectedProduct.id
+            },
+
```
Comment (kenneth-bakke) [inline client/src/components/ratingsReviews/RatingsReviews.jsx:242]: Overall this structure looks nice and a bit easier to read. Well done!

**Review by kenneth-bakke — APPROVED** [review]
Looks good, I like the new pattern on the API calls. Of course they'll be different with the new server routing, but that aside it looks very clear and easy to understand.

File: client/src/components/ratingsReviews/RatingsReviews.jsx (hunk 2 above)
Comment (hathadam) [reply client/src/components/ratingsReviews/RatingsReviews.jsx:217]: got it! Will see the replacement after yr PR.

File: client/src/components/ratingsReviews/RatingsReviews.jsx (hunk 3 above)
Comment (hathadam) [reply client/src/components/ratingsReviews/RatingsReviews.jsx:242]: Thanks!

**Pull request merged**